ts/WoltLabSuite/Core/Ui/Object/Action/Toogle.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Reacts to objects being toggled.
+ *
+ * @author  Matthias Schmidt
+ * @copyright  2001-2021 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Object/Action/Toggle
+ */
+
+import * as Language from "../../../Language";
+import UiObjectActionHandler from "./Handler";
+import { DatabaseObjectActionResponse } from "../../../Ajax/Data";
+
+function toggleObject(data: DatabaseObjectActionResponse, objectElement: HTMLElement): void {
+  const toggleButton = objectElement.querySelector('.jsObjectAction[data-object-action="toggle"]') as HTMLElement;
+
+  if (toggleButton.classList.contains("fa-square-o")) {
+    toggleButton.classList.replace("fa-square-o", "fa-check-square-o");
+
+    const newTitle = toggleButton.dataset.disableTitle
+      ? toggleButton.dataset.disableTitle
+      : Language.get("wcf.global.button.disable");
```

Comment:
```suggestion
    const newTitle = toggleButton.dataset.disableTitle || Language.get("wcf.global.button.disable");
```

Reply:
`TS1336: An index signature parameter type cannot be a type alias. Consider writing '[key: string]: ClipboardItemData' instead.`